### PR TITLE
Fixes for MPU6000 and MatekH743-slim board

### PR DESCRIPTION
**Pull request by vvladic**
- Board: Fixed DRDY pin from PB1 to PB2
- MPU6000: Added 10us delay for R/W of slow registers

Please use [PX4 Discuss](http://discuss.px4.io/) or [Slack](http://slack.px4.io/) to align on pull requests if necessary. You can then open draft pull requests to get early feedback.

### CODE REVIEW

File: src/drivers/imu/invensense/mpu6000/MPU6000.cpp [hunk 1]
```diff
@@ -435,6 +437,7 @@ uint8_t MPU6000::RegisterRead(Register reg)
 	cmd[0] = static_cast<uint8_t>(reg) | DIR_READ;
 	set_frequency(SPI_SPEED); // low speed for regular registers
 	transfer(cmd, cmd, sizeof(cmd));
+	up_udelay(10);
```
Comment (dagar) [inline src/drivers/imu/invensense/mpu6000/MPU6000.cpp:440]:
```suggestion
	px4_udelay(10);
```

This allows the driver to be built and used on Linux.

File: src/drivers/imu/invensense/mpu6000/MPU6000.cpp [hunk 2]
```diff
@@ -443,6 +446,7 @@ void MPU6000::RegisterWrite(Register reg, uint8_t value)
 	uint8_t cmd[2] { (uint8_t)reg, value };
 	set_frequency(SPI_SPEED); // low speed for regular registers
 	transfer(cmd, cmd, sizeof(cmd));
+	up_udelay(10);
```
Comment (dagar) [inline src/drivers/imu/invensense/mpu6000/MPU6000.cpp:449]:
```suggestion
	px4_udelay(10);
```

**Review by dagar — DISMISSED** [review]

File: src/drivers/imu/invensense/mpu6000/MPU6000.cpp [hunk 3]
```diff
@@ -435,6 +437,7 @@ uint8_t MPU6000::RegisterRead(Register reg)
 	cmd[0] = static_cast<uint8_t>(reg) | DIR_READ;
 	set_frequency(SPI_SPEED); // low speed for regular registers
 	transfer(cmd, cmd, sizeof(cmd));
+	px4_udelay(10);
```
Comment (dagar) [inline src/drivers/imu/invensense/mpu6000/MPU6000.cpp:440]: Not that it really matters, but maybe putting the delay before the transfer makes slightly more sense?

Comment (vvladic) [reply src/drivers/imu/invensense/mpu6000/MPU6000.cpp:440]: Now that you mention it, yes, it makes slightly more sense

**Pull request merged**